host-bin/startandroidstudio

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+APPLICATION="${0##*/}"
+
+
+USAGE="$APPLICATION [options]
+Wraps enter-chroot to launch Android Studio in an xiwi window.
+By default, it will log into the primary user on the first chroot found.
+Options are directly passed to enter-chroot; run enter-chroot to list them."
+
+eval "exec sh -e "`dirname "\`readlink -f "$0"\`"`/enter-chroot" -t android-studio "$@" "" \
```

Comment:
Not sure why you added the eval commit; you don't need it (and it screws up your quoting)